itest/test/bugs.test.ts

```diff
@@ -1,28 +1,99 @@
 import BugzillaAPI from "../../src";
 
-test("Create bug", async () => {
-  let api = new BugzillaAPI(
+let api: BugzillaAPI;
+
+const bug = {
+  product: "TestProduct",
+  component: "TestComponent",
+  summary: "A test bug",
+  version: "unspecified",
+  description: "This is a test bug",
+  op_sys: "Mac OS",
+  platform: "Macintosh",
+  priority: "High",
+  severity: "major",
+};
+
+let bugs: number[] = [];
+
+beforeAll(async () => {
+  api = new BugzillaAPI(
     "http://localhost:8088/bugzilla/",
     "admin@nowhere.com",
     "adminpass",
   );
 
-  await expect(
-    api.createBug({
+  bugs.push(await api.createBug(bug));
+  bugs.push(await api.createBug(bug));
+  bugs.push(
+    await api.createBug({
       product: "TestProduct",
       component: "TestComponent",
-      summary: "A test bug",
+      summary: "A test bug 2",
       version: "unspecified",
-      description: "This is a test bug",
+      description: "This is a test bug 2",
       op_sys: "Mac OS",
       platform: "Macintosh",
       priority: "High",
-      severity: "major",
+      severity: "normal",
     }),
-  ).resolves.toBe(1);
+  );
+
+  expect(bugs).toBeDefined();
+  expect(bugs.length).toEqual(3);
+});
 
-  let bugs = await api.getBugs([1]);
-  expect(bugs).toEqual([
+test("Fetch bugs", async () => {
+  let result = await api.getBugs([bugs[0] as number, bugs[1] as number]);
```

Comment:
Doesn't seem like the `as number` should be needed?

Reply:
I used type casting because otherwise I was getting error:
```
Type 'number | undefined' is not assignable to type 'string | number'.
  Type 'undefined' is not assignable to type 'string | number'.ts(2322)
```

Typescript is not able to detect that array won't be empty.